### ISSUE TITLE
Need for new forks and clones

### ISSUE THREAD

**Issue by jameshowison** [howisonlab/softcite-dataset#221]
I think it's time to have students create new forks and clones (after the file location changes, etc).  I used sudo to remove the pdf-files from everyone's working directories, but when they git pull their we can really cut things down by using

git clone --depth = 1 <repo_url>

which doesn't pull down full history for each student's working directory.  @jlcohoon, could you test that with a two new repos in your working directory, make a pull request, accept it, then git pull upstream into the second repo?

If that all works can you update the documentation to use --depth=1 wherever we clone.

**Comment by jlcohoon** [comment]
Will do this today!

**Comment by jlcohoon** [comment]
It worked. The pdf files got deleted and I didn't have any problems. I've updated the instructions (see step 9 here: https://howisonlab.github.io/softcite-dataset/secondDay.html#fork-and-clone-the-softcite-repo). 

On Friday should we have everyone delete their forks and make new ones?

**Comment by jameshowison** [comment]
Yes, if not before :)

**Comment by softcite-tester** [comment]
Ok, I re-did the history pdf removal; there were a bunch of large PDF files still in the history (as well as the csv file with all the open access files). Not sure what happened there, might have been added after I did the initial removal? Anyway, they are gone now).

I slimmed the softcite-dataset repo down to under 20M which makes much more sense (still have full history outside the docs/pdf-files directory, and full history on those files in the softcite-pdf-files repo).

The forks do still have the full history, so they will need to be deleted and re-created, then re-cloned.  I think that one could force a push to them, but let's avoid that complexity.

Work could be checked in first but would be easier to just pull it from the backup of the old, large, repo after creating the new forks, bringing across any of the work files that somehow hadn't made it out of the student's forks. ie have the students make a new fork and clone and then assess whether something of theirs is missing.
